spring-xd-spark-streaming/src/main/java/org/springframework/xd/spark/streaming/java/SparkStreamingModuleExecutor.java

```diff
@@ -45,13 +39,8 @@
 
 	private static SparkMessageSender messageSender;
 
-	private static final String HEADER_PREFIX = "spark.";
-
-	// todo: allow this to be overridden for any Spark Streaming Processor
-	private final MessageConverter converter = new DefaultSparkStreamingMessageConverter();
-
 	@SuppressWarnings("rawtypes")
-	public void execute(JavaDStream input, Processor processor, final SparkMessageSender sender) {
+	public void execute(JavaDStreamLike input, Processor processor, final SparkMessageSender sender) {
```

Comment:
It would still require casting where it's used, so it's probably not worth the effort anyways. So, nevermind.